OPERATOR.rst

```diff
@@ -221,6 +219,7 @@ Backporting from ``prod`` to ``develop``
 
 **Note that the HEAD from the merge commit needs to be the same as the HEAD commit on the PR branch.**
 
+.. _#team-boardwalk Slack channel: https://ucsc-gi.slack.com/archives/C705Y6G9Z
 
```

Comment:
Should this have two blank lines before a section like the UPGRADING.rst?

Reply:
I don't care to add it, but if it's important to you, feel free.